_posts/dart-workshop-01-2025-response.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "DATA Statement on Upcoming DART Board Workshops on January 8th and 10th 2025"
+excerpt: "The upcoming DART Board working sessions are expected to discuss key decision points regarding the governance and finances of DART"
+coverImage: "/assets/blog/dart-workshop-01-25-response/cover.png"
+date: "2025-01-07T05:00:00.000Z"
+author:
+  name: DATA Alert
+  picture: "/assets/blog/authors/data-team.png"
+ogImage:
+  url: "/assets/blog/dart-workshop-01-25-response/cover.png"
+banner: 
+  text: "ACTION REQUESTED: EMAIL YOUR DART BOARD ABOUT THE GOVERNANCE OF DART"
+---
+
+## Contact your DART board members!
+
+[https://dart.org/about/about-dart/board-members](https://dart.org/about/about-dart/board-members)
+
+## Tell the DART Board about your opposition to funding caps and cuts
```

Comment:
Should there be a different call to action here? I assume this is copied from the previous one

Reply:
yes, updated the cta